.dockerignore

```diff
@@ -18,6 +18,7 @@ hack/
 docs/
 examples/
 .github/
+!test/fixture
```

Comment:
I think you will also need to add `!test/container` (see test/container/dockerfile:90).